db/migrate/20160830233154_add_port_ranges_to_load_balancer_listener.rb

```diff
@@ -0,0 +1,8 @@
+class AddPortRangesToLoadBalancerListener < ActiveRecord::Migration[5.0]
+  def change
+    change_table :load_balancer_listeners do |t|
+      t.column :load_balancer_port_range, :int4range, :default => nil, :null => true
+      t.column :instance_port_range, :int4range, :default => nil, :null => true
```

Comment:
Nope, looks like it was [introduced in 9.2](https://www.postgresql.org/docs/9.2/static/rangetypes.html) so I'm fine with it.